assets/i18n/strings/compile.js

```diff
@@ -57,7 +57,7 @@ async function main() {
  */
 
 // NOTE: Please don't edit this file directly!
-//   You can (re)compile this file by running the "yarn build-i18n" shell command.
+//   This document describes how to change i18n strings in swg-js: https://docs.google.com/document/d/1FMEKJ_TmjHhqON0krE4xhDbTEj0I0DnvzxMzB2cWUWA/edit?resourcekey=0-TQ7hPOzAD4hX8x9PfweGSg#heading=h.q9gi7t4h1tyj
```

Comment:
Is this document available externally as well? Would this be better as a README or other doc posted on Github?

Reply:
This document is available internally to Googlers, since the strings and translations are stored internally, then copied over to this GitHub repository